File: php/Petshop.php
<?php
class Petshop {
    protected $ID;
    protected $nama;
    protected $kategori;
    protected $harga;
    protected $gambar_produk;

    public function __construct($ID, $nama, $kategori, $harga, $gambar_produk) {
        $this->ID = $ID;
        $this->nama = $nama;
        $this->kategori = $kategori;
        $this->harga = $harga;
        $this->gambar_produk = $gambar_produk;
    }

    public function getID() {
        return $this->ID;
    }

    public function getNama() {
        return $this->nama;
    }

    public function getKategori() {
        return $this->kategori;
    }

    public function getHarga() {
        return $this->harga;
    }

    public function getGambarProduk() {
        return $this->gambar_produk;
    }
}
?>

File: python/Petshop.py
class Petshop:
    def __init__(self, ID=None, nama=None, kategori=None, harga=None):
        self._ID = ID              
        self._nama = nama          
        self._kategori = kategori
        self._harga = harga

    # Getter methods
    def getID(self):
        """Fungsi untuk mendapatkan ID produk"""
        return self._ID

    def getNama(self):
        """Fungsi untuk mendapatkan nama produk"""
        return self._nama

    def getKategori(self):
        """Fungsi untuk mendapatkan kategori produk"""
        return self._kategori

    def getHarga(self):
        """Fungsi untuk mendapatkan harga produk"""
        return self._harga

    # Setter methods
    def setID(self, ID):
        """Fungsi untuk menetapkan ID produk"""
        self._ID = ID

    def setNama(self, nama):
        """Fungsi untuk menetapkan nama produk"""
        self._nama = nama

    def setKategori(self, kategori):
        """Fungsi untuk menetapkan kategori produk"""
        self._kategori = kategori

    def setHarga(self, harga):
        """Fungsi untuk menetapkan harga produk"""
        self._harga = harga